### ISSUE TITLE
spam account?

### ISSUE THREAD

**Issue by nschloe** [pypa/packaging-problems#501]
I just discovered [an account](https://pypi.org/user/airdge/) on PyPI that registered a number of packages (curl, chanel, dior, etc.), uploading the same "empty" code for each one of them. Looks like spam.

**Comment by merwok** [comment]
I think there’s a better place to report this, but I can’t find it (searched in packaging guide and PyPI footer links).

**Comment by merwok** [comment]
Tagging @ewdurbin as PyPI admin

**Comment by nschloe** [comment]
> (searched in packaging guide and PyPI footer links).

Yeah same. Let me know if you find something else and I'll go there.
